types/jquery.appear/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for JQuery Appear 1.0
+// Project: https://github.com/bas2k/jquery.appear
+// Definitions by: Anderson Friaça <https://github.com/AndersonFriaca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+declare namespace JQueryAppear {
```

Comment:
Instead of adding a new namespace, you could `export interface Options` directly, and wrap the augmentation of jquery in `declare global { interface JQuery { ... } }`.

Reply:
For readability, it is't better with namespace?

Reply:
Better to avoid polluting the global scope -- a module with direct `export` statements resolves based on file location, so it doesn't introduce any globals.

Reply:
Another thing you could do is inline the definition of `Options`: `options?: { data?: T, ... }`

Reply:
Thanks for explanation